### PR TITLE
adding more information/consistency around how to find device IDs

### PR DESCRIPTION
**Pull request by jvaleski**
## :recycle: Current situation

deviceID usage and definition is a bit scattered and confusing.

## :bulb: Proposed solution

adding a little bit of clarity around where to obtain the deviceID during config/setup

## :gear: Release Notes

just a couple of documentation changes.

### Testing

*Which tests were added? Which existing tests were adapted/changed? Which situations are covered, and what edge cases are missing?*

### Reviewer Nudging

*Where should the reviewer start? what is a good entry point?*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,11 +44,7 @@
 - ### If using BLE Connection
   1. Download SwitchBot App on App Store or Google Play Store
   2. Register a SwitchBot account and log in into your account
-  3. Click on Device wanting to connect too plugin
-     - Click the Settings Gear
-     - Click Device Info
-     - Copy BLE Mac aka `deviceId`
-  4. Input your `deviceId` into the Device Config
+  3. Input your `deviceId` into the Device Config
```
Comment (donavanbecker) [inline README.md:47]: Thanks for the PR!

We would need to flag this as optional, as it is not required but helpful if you want to override defaults.

**Review by donavanbecker — CHANGES_REQUESTED** [review]

**Comment by donavanbecker** [comment]
@jvaleski Feel free to reopen this and point it to the new beta branch.
